build.cake

```diff
@@ -131,25 +150,44 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(UNIT_TEST_PATH);
+		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
 	});
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
-Task("Package")
+Task("PackageNuGet")
 	.IsDependentOn("Build")
 	.Does(() => 
 	{
-		CreateDirectory(PACKAGE_DIR);
-
-        NuGetPack(NUSPEC_PATH, new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = BIN_DIR,
-            OutputDirectory = PACKAGE_DIR
-        });
+        NuGetPack("nuget/nunit-project-loader.nuspec",
+			new NuGetPackSettings()
+			{
+				Version = nugetVersion ?? packageVersion,
+				OutputDirectory = OUTPUT_DIR,
+				Files = new [] {
+					new NuSpecContent { Source = "../LICENSE.txt" },
```

Comment:
Is this the right path for Licence.txt? (https://github.com/nunit/nunit-project-loader/issues/7) Surprised Cake doesn't flag the missing file. 😞 

It would be good to have this as a rooted path for the avoidance of doubt, e.g. `Path.Combine(PROJECT_DIR, LICENSE.txt)`